README.md

```diff
@@ -64,6 +64,34 @@ Please install it and then accept the invitation received on your email address.
 
 The application can be downloaded from here: [Hubstaff Tracker](https://app.hubstaff.com/download)
 
+#### INTERNAL projects
+Below you can find the list of INTERNAL projects alongside with their descriptions.
+Descriptions tell you where you should bill your INTERNAL time on hubstaff.
+
+| Project                                          | Description                                                                                                                                           |
+| -----------------------------------------------  | ----------------------------------------------------------------------------------------------------------------------------------------------------  |
+| INTERNAL / content marketing                     | writing articles or blog posts under company name; presenting company in the outside world;                                                           |
+| INTERNAL / django application dockerization      | modifying our internal django docker template project;                                                                                                |
+| INTERNAL / finding new partners                  | time spent on finding new partners/customers;                                                                                                         |
+| INTERNAL / internal infrastructure management    | anything related to preparing your workspace to work; creating encrypted partitions, installing vms or any software needed to work;                   |
+| INTERNAL / management                            | activity reserved for company management;                                                                                                             |
+| INTERNAL / non-project meeting                   | phone calls, slack conversations which are related to your work  but not directly project related;                                                    |
+| INTERNAL / Ogarniacz                             | stuff related to organizing things needed in the company (like laptops or any needed  office stuff or hardware);                                      |
+| INTERNAL / opensource development                | time spent on looking for any opensource bugs to be solved or projects to take part in as well as actual programming, resolving issues;               |
+| INTERNAL / other internal development            | spending time on improving internal infrastructure like onboarding, training or any other company related stuff which might be helpful for others;    |
+| INTERNAL / pre-sales                             | any pre-work needed to be done before starting project with customer; usually asked by @ppolewicz;                                                    |
```

Comment:
before the contract is signed, not before starting project. Someone could interpret this as environment preparation.